evans with tls

HI,
How to use evans with tls? My gRPC server is configured to use tls, and i'm trying like this:

> `evans --tls --cert ca.crt --certkey ca.key -r --verbose 
evans: load global config from /home/sergiofm/.config/evans
evans: local config is not found
evans: bind flagset to the loaded config
evans: the conclusive config: &config.Config{
  Default: &config.Default{
    ProtoPath: []string{},
    ProtoFile: []string{},
    Package:   "",
    Service:   "",
  },
  Meta: &config.Meta{
    ConfigVersion: "0.8.0",
    AutoUpdate:    false,
    UpdateLevel:   "patch",
  },
  REPL: &config.REPL{
    PromptFormat:      "{package}.{sevice}@{addr}:{port}",
    InputPromptFormat: "{ancestor}{name} ({type}) => ",
    ColoredOutput:     true,
    Silent:            false,
    SplashTextPath:    "",
    HistorySize:       100,
  },
  Server: &config.Server{
    Host:       "127.0.0.1",
    Port:       "50051",
    Reflection: true,
    TLS:        true,
    Name:       "",
  },
  Log: &config.Log{
    Prefix: "evans: ",
  },
  Request: &config.Request{
    Header: {
      "grpc-client": []string{
        "evans",
      },
    },
    Web:         false,
    CACertFile:  "",
    CertFile:    "ca.crt",
    CertKeyFile: "ca.key",
  },
}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x9b0796]

goroutine 1 [running]:
github.com/ktr0731/evans/idl/proto.LoadByReflection(0x0, 0x0, 0x0, 0x0, 0x0, 0x6)
	/go/src/github.com/ktr0731/evans/idl/proto/proto.go:155 +0x26
github.com/ktr0731/evans/mode.newSpec(0xc000228450, 0x0, 0x0, 0x1, 0x1, 0xc0002d0c60, 0x50)
	/go/src/github.com/ktr0731/evans/mode/common.go:16 +0x7c
github.com/ktr0731/evans/mode.RunAsREPLMode(0xc000228450, 0xd68a80, 0xc0002cacf0, 0xc000294a00, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/mode/repl.go:28 +0x102
github.com/ktr0731/evans/app.(*App).run(0xc00000dee0, 0xc000030090, 0x7, 0x7, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/app/app.go:123 +0x453
github.com/ktr0731/evans/app.(*App).Run(0xc00000dee0, 0xc000030090, 0x7, 0x7, 0xc00000dec0)
	/go/src/github.com/ktr0731/evans/app/app.go:40 +0x4d
main.main()
	/go/src/github.com/ktr0731/evans/main.go:11 +0xc6
`

Hi, 

If i had CACertFile i get the same error:

 evans --tls --cert certs/server.crt --certkey certs/server.key --cacert certs/server.csr --host localhost -r --verbose
evans: load global config from /home/sergiofm/.config/evans
evans: migrated the global config to the structure of the latest version
evans: local config is not found
evans: bind flagset to the loaded config
evans: the conclusive config: &config.Config{
  Default: &config.Default{
    ProtoPath: []string{},
    ProtoFile: []string{},
    Package:   "",
    Service:   "",
  },
  Meta: &config.Meta{
    ConfigVersion: "0.8.0",
    AutoUpdate:    false,
    UpdateLevel:   "patch",
  },
  REPL: &config.REPL{
    PromptFormat:      "{package}.{sevice}@{addr}:{port}",
    InputPromptFormat: "{ancestor}{name} ({type}) => ",
    ColoredOutput:     true,
    Silent:            false,
    SplashTextPath:    "",
    HistorySize:       100,
  },
  Server: &config.Server{
    Host:       "localhost",
    Port:       "50051",
    Reflection: true,
    TLS:        true,
    Name:       "",
  },
  Log: &config.Log{
    Prefix: "evans: ",
  },
  Request: &config.Request{
    Header: {
      "grpc-client": []string{
        "evans",
      },
    },
    Web:         false,
    CACertFile:  "certs/server.csr",
    CertFile:    "certs/server.crt",
    CertKeyFile: "certs/server.key",
  },
}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9fd9f8]

goroutine 1 [running]:
github.com/ktr0731/evans/mode.RunAsREPLMode(0xc0002581b0, 0xd68a60, 0xc0002c8780, 0xc0001c9630, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/mode/repl.go:23 +0x88
github.com/ktr0731/evans/app.(*App).run(0xc0000a7ee0, 0xc0000c8010, 0xb, 0xb, 0x0, 0x0)
	/go/src/github.com/ktr0731/evans/app/app.go:123 +0x453
github.com/ktr0731/evans/app.(*App).Run(0xc0000a7ee0, 0xc0000c8010, 0xb, 0xb, 0xc0000a7ec0)
	/go/src/github.com/ktr0731/evans/app/app.go:40 +0x4d
main.main()
	/go/src/github.com/ktr0731/evans/main.go:11 +0xc6

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,6 @@ go 1.12
 
 require (
 	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
-	github.com/c-bata/go-prompt v0.2.3
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
 	github.com/fatih/color v1.7.0
 	github.com/golang/protobuf v1.3.1

--- a/grpc/grpc.go
+++ b/grpc/grpc.go
@@ -116,7 +116,7 @@ func NewClient(addr, serverName string, useReflection, useTLS bool, cacert, cert
 			}
 			cp := x509.NewCertPool()
 			if !cp.AppendCertsFromPEM(b) {
-				return nil, errors.Wrap(err, "failed to append the client certificate")
+				return nil, errors.New("failed to append the client certificate")
 			}
 			tlsCfg.RootCAs = cp
 		}

